src/tests/payments.js

```diff
@@ -0,0 +1,277 @@
+(function() {
+
+var assert = require('assert');
+var eq_ = assert.eq_;
+var contains = assert.contains;
+var mock = assert.mock;
```

Comment:
I agree it's a poor name. Just following the pattern of the tests already here and since that's the function name already I guess it's worth keeping.